### PR TITLE
Complete the Index File for the v6.4.0 Release Notes

### PR DESCRIPTION
**Pull request by cailafinn**
**Description of work.**
Created the main page for the upcoming release notes. Provides an overview of the major changes in the release and informs users of the packaging changes that we've made.

**To test:**
1. Delete the `doctrees` dir
2. Build the docs
3. Check there are no warnings
4. Check all links work
5. Grammar and spelling checks

*There is no associated issue.*

*This does not require release notes* because **it's about release notes**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.4.0/index.rst [hunk 1]
```diff
@@ -4,19 +4,50 @@
...
 
-**TODO: Add paragraph summarizing big changes**
+Many of the updates in this version have focused on improvements for users, building on existing algorithms and interfaces.
+Our method of packaging has also changed. Windows and macOS users should not notice any difference,
+but this will be the last release that uses the linux ``.rpm`` and ``.deb`` packages (more details :ref:`here <6.4_packaging>`).
+
+In addition to many improvements we are delighted to announce some new features including:
+
+- A new :ref:`cut viewer tool <sliceviewer_nonaxiscuts>` for non-axis aligned cuts has been added to the :ref:`SliceViewer`.
+- Workbench (and mslice) have been updated to be compatible with matplotlib v3.5.
```
Comment (sf1919) [inline docs/source/release/v6.4.0/index.rst:24]:
```suggestion
- Workbench (and MSlice) have been updated to be compatible with matplotlib v3.5.
```

File: docs/source/release/v6.4.0/index.rst [hunk 2]
```diff
@@ -4,19 +4,50 @@
...
+
+Packaging & Installation
+------------------------
+
+As advertised in :ref:`v6.3.0 <v6.3.0>` we have reworked our installer packages for this release. Changes include:
+
+- Windows and macOS packages are now named ``MantidWorkbench-6.4.0`` with the appropriate extension. Other than this they are used in the same manner.
+- A new linux package,``mantidworkbench-6.4.0.tar.xz``, now supports many Linux distributions and should work on any 2010+ distribution with ``glibc >= 2.17``.
+  It is a simple, flat tarball that can be unpacked anywhere and no longer has to reside in ``/opt``.
+- All of the above packages are built using our conda libraries to make future maintenance easier.
```
Comment (sf1919) [inline docs/source/release/v6.4.0/index.rst:40]: I'm wondering whether this sentence doesn't need to be a bullet point but a sentence after the bullet point list. Does that make sense?

**Comment by robertapplin** [comment]
Sorry @sf1919 , I assigned this before I saw your comments so not sure if I'm stepping on your toes.

The grammar looks good, there are no build warnings, and I've tested all the links. The only link that doesn't work is to the Github release page (https://github.com/mantidproject/mantid/releases/tag/v6.4.0) but I assume this will start working once the v6.4.0 tag has been published

I'm happy to approve this, if @sf1919 is happy?

**Comment by sf1919** [comment]
I was just doing a quick review of spelling grammar to help things along. We need @martyngigg to give it a quick read but if he's happy with the wording then you can approve it. Thanks @robertapplin  for doing a thorough test.

File: docs/source/release/v6.4.0/index.rst [hunk 3]
```diff
@@ -4,19 +4,51 @@
...
    :local:
 
-.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
-
 We are proud to announce version 6.4.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+Many of the updates in this version have focused on improvements for users, building on existing algorithms and interfaces.
+Our method of packaging has also changed. Windows and macOS users should not notice any difference,
+but this will be the last release that uses the linux ``.rpm`` and ``.deb`` packages (more details :ref:`here <6.4_packaging>`).
```
Comment (martyngigg) [inline docs/source/release/v6.4.0/index.rst:19]: This is slightly picky but could we reword this to: 

_but this will be the last release that we provide ``.rpm`` and ``.deb`` packages for Linux (more details :ref:`here <6.4_packaging>`)._

**Review by martyngigg — CHANGES_REQUESTED** [review]
Apart from the slight tweak to the initial paragraph I think this is good.

**Review by martyngigg — APPROVED** [review]
